plat/common/bootinfo.c

```diff
@@ -6,6 +6,12 @@
 
 #include <uk/essentials.h>
 #include <uk/plat/common/bootinfo.h>
+#include <uk/plat/common/sections.h>
+#ifdef CONFIG_LIBFDT
+#include <libfdt.h>
+#endif /* CONFIG_LIBFDT */
+
+#define ukplat_bootinfo_crash(s)		ukplat_crash()
```

Comment:
Just a wrapper like you would see in the initial Firecracker/Multiboot/UEFI code. Since console is not initialized at that point, I thought I would align this code with what has already been written for what would be executed before the console is initialized.

Reply:
Okay, thanks for clarifying